client/packages/coldchain/src/Sensor/Components/EditableSensorTab.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from 'react';
+import { SensorFragment } from '../api';
+import { Grid } from '@mui/material';
+import {
+  BasicTextInput,
+  InputWithLabelRow,
+  InputWithLabelRowProps,
+} from '@common/components';
+
+import { LocationSearchInput } from '@openmsupply-client/system/src';
+import { useTranslation } from '@common/intl';
+
+const StyledInputRow = ({ label, Input }: InputWithLabelRowProps) => (
+  <InputWithLabelRow
+    label={label}
+    Input={Input}
+    labelProps={{ sx: { textAlign: 'end', padding: '4' } }}
+    labelWidth="140px"
+    sx={{
+      padding: '4',
+      gap: '46px',
```

Comment:
There are a few examples like so where the CSS values feel a little arbitrary to get everything to line up.

Not sure if there is an easier way to manage the components here.